src/Robo/Commands/Setup/AllCommand.php

```diff
@@ -42,7 +42,10 @@ public function setup() {
         break;
     }
 
-    $commands[] = 'blt:init:shell-alias';
+    $commands = [
+      'blt:init:shell-alias',
+      'blt:init:drush:cli-tools'
```

Comment:
Won't this cause `vendor/bin/drush` to be removed locally? How will BLT be able to execute drush commands? I don't see any indication of BLT being aware of `composer-bin` and the `drush8` and `drush9` aliases for `DrushTask` executions.